From 2409a888fe4b45aeb8438d586d86b92e218ad18e Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 21 Jan 2019 16:24:12 -0700
Subject: [PATCH 03/25] ARM: configs: Disable CONFIG_CRYPTO_FIPS for hlte
 devices

* For some reason, decryption magically started failing with
  "crypto_lookup failed due to FIPS error: eseqiv". We
  can go without.

Change-Id: I6cf5c6265cddfa63b8a761ad73b0fc43fedae21d
Signed-off-by: Kevin F. Haggerty <haggertk@lineageos.org>
---
 arch/arm/configs/lineage_hlte_bcm2079x_defconfig | 2 +-
 arch/arm/configs/lineage_hlte_pn547_defconfig    | 2 +-
 arch/arm/configs/lineage_hltechn_defconfig       | 2 +-
 arch/arm/configs/lineage_hltekor_defconfig       | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/lineage_hlte_bcm2079x_defconfig b/arch/arm/configs/lineage_hlte_bcm2079x_defconfig
index 1d67530002d4..85b725059a54 100644
--- a/arch/arm/configs/lineage_hlte_bcm2079x_defconfig
+++ b/arch/arm/configs/lineage_hlte_bcm2079x_defconfig
@@ -4445,7 +4445,7 @@ CONFIG_CRYPTO=y
 #
 # Crypto core or helper
 #
-CONFIG_CRYPTO_FIPS=y
+# CONFIG_CRYPTO_FIPS is not set
 CONFIG_CRYPTO_FIPS_INTEG_COPY_ADDRESS=0x8A008000
 CONFIG_CRYPTO_ALGAPI=y
 CONFIG_CRYPTO_ALGAPI2=y
diff --git a/arch/arm/configs/lineage_hlte_pn547_defconfig b/arch/arm/configs/lineage_hlte_pn547_defconfig
index 1c33fd1e46dd..e66efd49127c 100644
--- a/arch/arm/configs/lineage_hlte_pn547_defconfig
+++ b/arch/arm/configs/lineage_hlte_pn547_defconfig
@@ -4446,7 +4446,7 @@ CONFIG_CRYPTO=y
 #
 # Crypto core or helper
 #
-CONFIG_CRYPTO_FIPS=y
+# CONFIG_CRYPTO_FIPS is not set
 CONFIG_CRYPTO_FIPS_INTEG_COPY_ADDRESS=0x8A008000
 CONFIG_CRYPTO_ALGAPI=y
 CONFIG_CRYPTO_ALGAPI2=y
diff --git a/arch/arm/configs/lineage_hltechn_defconfig b/arch/arm/configs/lineage_hltechn_defconfig
index b0b004c2e799..c1ceea508897 100644
--- a/arch/arm/configs/lineage_hltechn_defconfig
+++ b/arch/arm/configs/lineage_hltechn_defconfig
@@ -4446,7 +4446,7 @@ CONFIG_CRYPTO=y
 #
 # Crypto core or helper
 #
-CONFIG_CRYPTO_FIPS=y
+# CONFIG_CRYPTO_FIPS is not set
 CONFIG_CRYPTO_FIPS_INTEG_COPY_ADDRESS=0x8A008000
 CONFIG_CRYPTO_ALGAPI=y
 CONFIG_CRYPTO_ALGAPI2=y
diff --git a/arch/arm/configs/lineage_hltekor_defconfig b/arch/arm/configs/lineage_hltekor_defconfig
index 5254007b47be..0c3c3acc73fe 100644
--- a/arch/arm/configs/lineage_hltekor_defconfig
+++ b/arch/arm/configs/lineage_hltekor_defconfig
@@ -4445,7 +4445,7 @@ CONFIG_CRYPTO=y
 #
 # Crypto core or helper
 #
-CONFIG_CRYPTO_FIPS=y
+# CONFIG_CRYPTO_FIPS is not set
 CONFIG_CRYPTO_FIPS_INTEG_COPY_ADDRESS=0x8A008000
 CONFIG_CRYPTO_ALGAPI=y
 CONFIG_CRYPTO_ALGAPI2=y
-- 
2.17.1

